packages/shared/src/contexts/payment/ChromeExtension.tsx

```diff
@@ -0,0 +1,27 @@
+import type { PropsWithChildren, ReactElement } from 'react';
+import React from 'react';
+import { useRouter } from 'next/router';
+import { BasePaymentProvider } from './BasePaymentProvider';
+import type { OpenCheckoutProps } from './context';
+import { webappUrl } from '../../lib/constants';
+
+export const ChromeExtensionProvider = ({
```

Comment:
My main question was more if this one shouldn't also return the pricing id's?
Now it only exposes the openCheckout right

Reply:
It would return the pricing ids. The `BasePaymentContext` handles that, and everything common in terms of Payment for web.